src/phoenix/server/api/routers/v1/__init__.py

```diff
@@ -21,6 +21,16 @@
     Route("/v1/datasets", datasets.list_datasets, methods=["GET"]),
     Route("/v1/datasets/{id:str}", datasets.get_dataset_by_id, methods=["GET"]),
     Route("/v1/datasets/{id:str}/csv", datasets.get_dataset_csv, methods=["GET"]),
+    Route(
+        "/v1/datasets/{id:str}/jsonl/openai_ft",
+        datasets.get_dataset_jsonl_openai_ft,
+        methods=["GET"],
+    ),
+    Route(
+        "/v1/datasets/{id:str}/jsonl/openai_evals",
+        datasets.get_dataset_jsonl_openai_evals,
+        methods=["GET"],
+    ),
```

Comment:
Are we doing underscores or dashes?

Reply:
underscore for urls i think (dashes for params?)